### ISSUE TITLE
Cannot read property 'click' of undefined

### ISSUE THREAD

**Issue by serjo96** [kidjp85/react-id-swiper#115]
Hello!

I hope you can help me, I get this error every time the component is unmounted

Error leads to code swiper.js and audit of listeners
https://yadi.sk/i/OsKluWDH3QAKQb

Erorr take in check `if (self.eventsListeners[event]) {`

What am I doing: 1) I get data from the server and when they are received I add value to show on ture, and show the component with the slider, if there are not any, then I do not show it
2) When componentWillUnmount or when I get new data, I set the property to check the rendering value to false

```
if (this.props.tv.isFetching) {
	        return (
<div className="recommendations__list tooltip-parent">
            <Swiper {...SwiperParam}
                shouldSwiperUpdate={true}
                containerClass={ `swiper-container ${movie.recommendations.total_results < 10 ? 'slider-container--centered' : null}`}
                navigation={movie.recommendations.total_results > 10 ? {
		            nextEl: '.slider-button-next',
		            prevEl: '.slider-button-prev',
		            disabledClass: 'slider-button--isDisabled'
                } : {
	                nextEl: null,
	                prevEl: null,
                }}
            >
                {movie.recommendations.results.map((el, index)=>
                    (<div key={index}>
                        <MovieItem
                            title={el.title || el.name}
                            original_title={el.original_title || el.original_name}
                            overview={el.overview}
                            voteAverage={el.vote_average}
                            date={el.release_date || el.first_air_date}
                            poster={el.poster_path}
                            id={el.id}
                            typeList={movie.typeList}
                        />

                    </div>)
                )}
            </Swiper>
        </div>
)
}
```
and code when erorr be

```
SwiperClass.prototype.emit = function emit () {
    var args = [], len = arguments.length;
    while ( len-- ) args[ len ] = arguments[ len ];

  var self = this;
  var events;
  var data;
  var context;
  if (typeof args[0] === 'string' || Array.isArray(args[0])) {
    events = args[0];
    data = args.slice(1, args.length);
    context = self;
  } else {
    events = args[0].events;
    data = args[0].data;
    context = args[0].context || self;
  }
  var eventsArray = Array.isArray(events) ? events : events.split(' ');
  eventsArray.forEach(function (event) {
    if (self.eventsListeners[event]) {
      var handlers = [];
      self.eventsListeners[event].forEach(function (eventHandler) {
        handlers.push(eventHandler);
      });
      handlers.forEach(function (eventHandler) {
        eventHandler.apply(context, data);
      });
    }
  });
  return self;
};
```

**Comment by kidjp85** [comment]
Hi @serjo96 , can u try newest version. It seems this error is from Swiper, and it should be fixed with new ver 1.5.6. 
https://github.com/nolimits4web/Swiper/issues/2304

**Comment by serjo96** [comment]
Yea, thanks, now all work!
